modules/cco-ccoctl-install-creating-manifests.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// AWS assemblies:
+// * installing/installing_aws/installing-aws-customizations.adoc
+// * installing/installing_aws/installing-aws-network-customizations.adoc
+// * installing/installing_aws/installing-restricted-networks-aws-installer-provisioned.adoc
+// * installing/installing_aws/installing-aws-vpc.adoc
+// * installing/installing_aws/installing-aws-private.adoc
+// * installing/installing_aws/installing-aws-government-region.adoc
+// * installing/installing_aws/installing-aws-secret-region.adoc
+// * installing/installing_aws/installing-aws-china.adoc
+// * installing/installing_aws/installing-aws-localzone.adoc
+// * installing/installing_aws/installing-aws-outposts-remote-workers.adoc
+//
+// GCP assemblies:
+// * installing/installing_gcp/installing-gcp-customizations.adoc
+// * installing/installing_gcp/installing-gcp-network-customizations.adoc
+// * installing/installing_gcp/installing-restricted-networks-gcp-installer-provisioned.adoc
+// * installing/installing_gcp/installing-gcp-vpc.adoc
+// * installing/installing_gcp/installing-gcp-shared-vpc.adoc
+// * installing/installing_gcp/installing-gcp-private.adoc
+
+:_content-type: PROCEDURE
+[id="cco-ccoctl-install-creating-manifests_{context}"]
+= Clusters configured to use short-term credentials only: Incorporating the Cloud Credential Operator utility files
+
+To implement short-term, limited-privilege security credentials for individual components, you must move the files that the CCO utility (`ccoctl`) creates to the correct directories for the installation program to use them.
+
+.Prerequisites
+
+* You have configured an account with the cloud platform that hosts your cluster.
+* You have completed the following steps under "Configuring a cluster to use short-term credentials" in the "Manually creating IAM" chapter for your provider:
+** You have configured the Cloud Credential Operator utility (`ccoctl`).
+** You have created the cloud provider resources that are required for your cluster with the `ccoctl` utility.
+* You have set the `credentialsMode` parameter to `Manual` in the `install-config.yaml` configuration file.
```

Comment:
Instead of listing this as a prerequisite, consider adding a step to the procedure to tell users to set `credentialsMode` to `Manual`.

See my comment in the sample install config modules for AWS and GCP.